apps/example/package.json

```diff
@@ -50,51 +50,51 @@
     "@sentry/react-native": "^5.36.0",
     "@statsig/react-native-bindings": "^3.17.1",
     "@walletconnect/react-native-compat": "^2.18.1",
-    "expo": "~52.0.47",
-    "expo-build-properties": "~0.13.0",
-    "expo-camera": "^16.0.18",
-    "expo-constants": "~17.0.4",
-    "expo-dev-client": "~5.0.0",
-    "expo-splash-screen": "~0.29.13",
-    "expo-status-bar": "~2.0.0",
+    "expo": "^53.0.22",
+    "expo-build-properties": "~0.14.8",
+    "expo-camera": "~16.0.18",
+    "expo-constants": "~17.1.7",
+    "expo-dev-client": "~5.2.4",
+    "expo-splash-screen": "~0.30.10",
```

Comment:
I remember we had stayed on the older version to keep the fullscreen gradient splash on iOS and/or Android.
So I guess upgrading will change the behavior and we may not have it anymore.
Not a huge deal. Just noting.
cc @bakoushin who was closer to this.